src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -768,15 +768,24 @@ protected DynamoModel(IStartConfiguration config)
         /// does not already exist in the list.
         /// </summary>
         /// <param name="path"> The path to add.</param>
-        public bool AddPackagePath(string path)
+        /// <param name="file"> The file to add when importing a library.</param>
+        public bool AddPackagePath(string path, string file)
```

Comment:
can it be `public bool AddPackagePath(string path, string file="")` ? i.e adding default param, so the other places doesn't need a change.